packages/blocks/src/api/parser.js

```diff
@@ -378,6 +378,6 @@ const createParse = ( parseImplementation ) =>
  *
  * @return {Array} Block list.
  */
-export const parseWithGrammar = createParse( grammarParse );
+export const parseWithGrammar = createParse( defaultParse );
```

Comment:
Should we offer a filter for JS implementation, too?

Reply:
yes but I wasn't sure if this PR was the right one for it. that is, filtering out the PHP side seemed somewhat straightforward while filtering the JS side seemed more complicated since we have to take into account things like loading the parser bundles and making sure they are available before the editor loads

do you think we need to do it all here in this PR?

Reply:
It's totally fine as its own PR, I just wanted to ensure we tackle both PHP and JS side of things.